Live location

danleedham opened the pull request:
[Dans Live Location.mov.zip](https://github.com/moschopsuk/Roses-2015-CasparCG-Graphics/files/937695/Dans.Live.Location.mov.zip)

The largest change is showcased within the video above. 

The main idea here was to separate the 'general' bug elements. The word for Live has become dynamic and the bugtext, which existed but wasn't accessible from the client, is also dynamic. A major change to the operational aspect of this is that now you can have one, two, or three (and any of the two together) without having to have the clock on. This is shown in the video. 

I also changed a bit of the CSS to make the overall package look less like LA1TV and more like ROSES BABY. 

What could possibly do with cleaning up - choosing either **General** OR **Bug** terms, as bug and general are both used on the same type of elements.

danleedham committed:
Made general Dynamic

Changed quite a few things:
- This is a drastic ‘look’ change from the red and black to the Roses
2017 colors
- Each general element can be added or taken off without needing the
others on
- ‘Live’ can be replaced with anything (default: Live)
- location cab be replaced with anything (default: York University)

danleedham committed:
Minor CSS changes

danleedham committed:
Minor CSS changes to grid

danleedham committed:
CSS

stephenrob commented on server.js:
```diff
@@ -6,7 +6,7 @@ var app = express();
 var io = require('socket.io').listen(server);
 
-var bug = {};
+var bug = {livetext: "Live", loctext: "York University"};
```
Can `loctext` be replaced with `locationtext` just makes it more readable.

danleedham replied:
Done.

stephenrob commented on public/admin/templates/general.tmpl.html:
```diff
@@ -4,12 +4,25 @@
+    <div class="sixteen wide column">
+      <div class="inline fields">
+  		<div class="five wide field">
+           <input name="bugtext" placeholder="Live Location" type="text" ng-model="bug.loctext" />
```
Can `loctext` be replaced with `locationtext` just makes it more readable

stephenrob commented on public/index.html:
```diff
@@ -12,8 +12,13 @@
+                 <div class="bug top scale-fade" ng-show="state.showLive">
+                 <h1 ng-show="state.showLive">{{bug.livetext}}</h1>
+                 </div>
+                 <div class="bug top scale-fade" ng-show="state.showLocation"><h1 class="white" ng-show="state.showLocation">{{bug.loctext}}</h1></div>
```
Can `loctext` be replaced with `locationtext` just makes it more readable

stephenrob commented on server.js:
```diff
@@ -6,7 +6,7 @@ var app = express();
 var io = require('socket.io').listen(server);
 
-var bug = {};
+var bug = {livetext: "Live", loctext: "York University"};
```
Doesn't seem right hard coding the default location to be `York University` as although correct for this year isn't generic. (Yes I know its an easy thing to change :) )

danleedham replied:
The reason I set this to something by default is so that someone can't put it live without it being set... maybe that's not too clever! Having looked at other dynamic variables, there are loads of elements which can go on blank. 

I'll unset this but is it worth looking at not allowing the graphic/element to show if the variable = ''?

stephenrob reviewed (changes requested):
All looks fine apart from some minor tweaks.

Haven't tested against caspar yet will do that tomorrow when on a machine with caspar.

danleedham committed:
Amendments as requested by @stephenrob made

Loctext changed to location text
Default value for location text removed. Previously read York
University.

danleedham committed:
Closed 22

done #22

danleedham committed:
Done #22

Hello Tom

TMannall commented:
Works fine in CasparCG Server 2.0.7 

![image](https://cloud.githubusercontent.com/assets/10275203/25202324/e0837242-254c-11e7-9a6f-c8794f424000.png)
